.github/workflows/check.yml

```diff
@@ -0,0 +1,27 @@
+on: [push, pull_request]
+name: CI Checks
+jobs:
+  check:
+    name: Check and Lint
+    runs-on: ubuntu-latest
```

Comment:
To avoid possible issues when a new Ubuntu LTS emerges, could this be `runs-on: ubuntu-20.04`?